arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1642,7 +1642,7 @@
         "tags": [
           "Subscriptions"
         ],
-        "operationId": "Subscriptions_ListAll",
+        "operationId": "Subscriptions_ListByTopic",
```

Comment:
@anudeepsharma they are called subscriptions as well in the portal. This subscription is a child resource of a topic.